utils/remote_control/navigator_emergency_control/nodes/nodes/navigator_emergency.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+
+
+from remote_control_lib import RemoteControl
+import rospy
+from sensor_msgs.msg import Joy
+
+
+__maintainer__ = "Anthony Olive"
```

Comment:
According to precedent, the author of the file is it's original creator, so I agree with Kevin here.